terraform/modules/configs/deploy-all/main.tf

```diff
@@ -551,3 +591,21 @@ module "client" {
   deployment_maximum_percent         = var.deployment_maximum_percent
   deployment_minimum_healthy_percent = var.deployment_minimum_healthy_percent
 }
+
+######################################
+# S3 Virus scan Service
+######################################
+
+module "s3_virus_scan" {
+  source                             = "../../services/s3-virus-scan"
+  environment                        = var.environment
+  vpc_id                             = data.aws_ssm_parameter.vpc_id.value
+  ecs_cluster_id                     = module.ecs_s3_virus_scan.ecs_cluster_id
+  private_app_subnet_ids             = split(",", data.aws_ssm_parameter.private_app_subnet_ids.value)
+  execution_role_arn                 = aws_iam_role.ecs_task_execution_role.arn
+  app_port                           = "4567"
+  cpu                                = var.s3_virus_scan_cpu
+  memory                             = var.s3_virus_scan_memory
+  ecr_image_id_s3_virus_scan         = var.ecr_image_id_s3_virus_scan
```

Comment:
This var needs adding to the module's `variables.tf`

Reply:
@tommyb82 this variable has been added [here](https://github.com/Crown-Commercial-Service/ccs-scale-infra-services-bat/blob/80464438f96843ab42c370d565256551142c3ee0/terraform/modules/configs/deploy-all/variables.tf#L34)

Reply:
Yep but it needs to be propagated down into the new `s3_virus_scan` module's variables too 👍